libraries/xcode.rb

```diff
@@ -9,9 +9,15 @@ def installed?(semantic_version)
         installed_xcodes.include?(semantic_version)
       end
 
-      def bundle_version_correct?
-        xcode_version = '/Applications/Xcode.app/Contents/version.plist CFBundleShortVersionString'
-        node['macos']['xcode']['version'] == shell_out("defaults read #{xcode_version}").stdout.strip
```

Comment:
FWIW, this could still be accessed (set in this case), with `Chef.node.default['macos']['xcode']['version']`. However, since this method wasn't even used, I don't think there's any reason to keep it. Down the road, we'll probably use something like this when we implement `load_current_value` and check the current version before doing anything.